### PR TITLE
Milestone 4: Portfolio Desktop version

### PR DESCRIPTION
**Pull request by Chandan-devs-tech**
## Project Requirements 

- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).

- In order to lay out the elements on the page you should use Grid and Flexbox.

- You don't need to implement any functionality that requires JavaScript.

## In this milestone I was able to :

- Learn and implement forms in HTML
- Learn how to use form spree as a free form data handler
- Practice my Github flow knowledge
- Practice CSS layouts with flexbox, grids, and media query.
- Learn how to do animation with CSS
- Practice pair programming

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,18 +93,18 @@ <h2 class="card-title">Tonic</h2>
           <button class="card-btn">See Project</button>
```
Comment (RaoAkif) [inline index.html:93]: Your button design and the hover transition is looking classy. But I think you forgot to add the `Button Pressed` design to it. Kindly add the color of the button on the `Pressed` state

FIGMA Design 
image

File: index.html [hunk 2]
```diff
@@ -81,18 +93,18 @@ <h2 class="card-title">Tonic</h2>
...
           <h2 class="card-title">
-            Multi-Post <br />
+            Multi-Post
             Stories
           </h2>
           <ul class="card-info">
-            <li class="bold-info">CANOPY</li>
-            <li class="info bullet"><span></span>Back End Dev</li>
+            <li class="bold-info">FACEBOOK</li>
+            <li class="info bullet"><span></span>Full Stack Dev</li>
```
Comment (RaoAkif) [inline index.html:107]: It seems the text is a little bit misaligned. Kindly take a look at it.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image | image  |

File: index.html [hunk 3]
```diff
@@ -112,10 +124,10 @@ <h2 class="card-title">
           <img src="Assets/Snapshoot Portfolio Card3.svg" alt="card1" />
         </div>
         <div class="card-content">
-          <h2 class="card-title">Tonic</h2>
+          <h2 class="card-title">Facebook 360</h2>
```
Comment (RaoAkif) [inline index.html:127]: The space between the text and the image seems quite small. Kindly take a look and try to match the design.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 4]
```diff
@@ -130,19 +142,18 @@ <h2 class="card-title">Tonic</h2>
...
-      <div class="card">
+      <div class="card-reverse">
         <div class="card-img">
           <img src="Assets/Snapshoot Portfolio Card4.svg" alt="card1" />
         </div>
         <div class="card-content">
           <h2 class="card-title">
-            Multi-Post <br />
-            Stories
+            Uber Navigation
```
Comment (RaoAkif) [inline index.html:151]: It seems the text is a little bit misaligned. Kindly take a look at it.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image | image  |

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @Chandan-devs-tech, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- You have done really impressive work. You took care of even minute things. Keep up the good work. 

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
### Hi  @Chandan-devs-tech , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


cr-approve

## Highlights

GitHub review feature :heavy_check_mark:
Inline comment :heavy_check_mark: 
The comments are given in a friendly and polite manner :heavy_check_mark: 
Explain why changes were suggested :heavy_check_mark: 

## Optional 
**_Every comment with the [**OPTIONAL**] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**